openfl-workspace/torch_cnn_mnist_straggler_check/plan/plan.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# Licensed subject to the terms of the separately executed evaluation license agreement between Intel Corporation and you.
+
+aggregator :
+  defaults : plan/defaults/aggregator.yaml
+  template : openfl.component.Aggregator
+  settings :
+    init_state_path     : save/torch_cnn_mnist_init.pbuf
+    best_state_path     : save/torch_cnn_mnist_best.pbuf
+    last_state_path     : save/torch_cnn_mnist_last.pbuf
+    rounds_to_train     : 6
+    log_metric_callback :
+      template : src.mnist_utils.write_metric
+
+
+collaborator :
+  defaults : plan/defaults/collaborator.yaml
+  template : openfl.component.Collaborator
+  settings :
+    delta_updates    : false
+    opt_treatment    : RESET
+
+data_loader :
+  defaults : plan/defaults/data_loader.yaml
+  template : src.ptmnist_inmemory.PyTorchMNISTInMemory
+  settings :
+    collaborator_count : 2
+    data_group_name    : mnist
+    batch_size         : 256
+
+task_runner :
+  defaults : plan/defaults/task_runner.yaml
+  template : src.pt_cnn.PyTorchCNN
+
+network :
+  defaults : plan/defaults/network.yaml
+
+assigner :
+  defaults : plan/defaults/assigner.yaml
+  
+tasks :
+  defaults : plan/defaults/tasks_torch.yaml
+
+compression_pipeline :
+  defaults : plan/defaults/compression_pipeline.yaml
+
+straggler_handling_policy :
```

Comment:
There should be a default template that would be copied if user ignored this section